cosmotech/coal/store/store.py

```diff
@@ -0,0 +1,61 @@
+import os
+import pathlib
+from typing import Optional
+
+import pyarrow
+import pyarrow.parquet as pq
+from adbc_driver_sqlite import dbapi
+from cosmotech.orchestrator.utils.singleton import Singleton
+
+from cosmotech.coal.utils.logger import LOGGER
+
+
+class Store(metaclass=Singleton):
+
+    @staticmethod
+    def sanitize_column(column_name: str) -> str:
+        return column_name.replace(" ", "_").lower()
+
+    def __init__(self, reset=False):
+        self.store_location = pathlib.Path(os.environ.get("CSM_PARAMETERS_ABSOLUTE_PATH", ".")) / ".coal/store"
+        self.store_location.mkdir(parents=True, exist_ok=True)
+        self._tables = dict()
+        self._database_path = self.store_location / "db.sqlite"
+        if reset and self._database_path.exists():
+            self._database_path.unlink()
+        self._database = str(self._database_path)
+
+    def get_table(self, table_name: str, columns: Optional[list[str]] = None) -> pyarrow.Table:
+        if not self.table_exists(table_name):
+            raise ValueError(f"No table with name {table_name} exists")
+        return self.execute_query(f"select * from {table_name}")
+        return pq.read_table(self._tables[table_name], columns=columns)
```

Comment:
Double *return*, i'm rusty but this should not work

Reply:
Forgot to remove the second return when I moved from parquet to sqlite